src/common/metrics/metrics.h

```diff
@@ -30,6 +30,13 @@ prometheus::Registry& GetMetricsRegistry();
 // This function should only be called by testing code.
 void TestOnlyResetMetricsRegistry();
 
+// A convenience wrapper to return a counter with the specified name and help message when
+// dimensions aren't known at compile time. This should only be used when dimensional data is
+// very low cardinality.
+inline auto& BuildRawCounter(const std::string& name, const std::string& help_message) {
```

Comment:
nit: let's use the prometheus terminology of families, so maybe call it `BuildCounterFamily`

Reply:
Sounds good and updated.